Carbon.IIS/Functions/Set-CIisAppPoolProcessModel.ps1

```diff
@@ -111,7 +119,14 @@ function Set-CIisAppPoolProcessModel
         return
     }
 
+    $targetMsg = 'IIS application pool defaults process model'

```

Comment:
Perhaps Invoke-SetConfigurationAttribute could take in, eg. `-Target @(message, scope)` , where scope is the identifier.